utility/pbkdf2-hash-generator/README.md

```diff
@@ -0,0 +1,12 @@
+# pbkdf2-hash-generator
+This contains the Client for generate PBKDF2 Hash and the Salt Value for a given plain text password.
+
+## How to use this Hash Generator
+
+1. Clone this repository.
+2. Build this module using `mvn clean install`
+3. Then the executable jar will be built inside the target folder.
```

Comment:
better to give the target folder path

Reply:
Addressed with https://github.com/wso2-extensions/identity-hash-provider-pbkdf2/pull/11/commits/219948b7973706042813c8241c3c026b40888070